pages/blog/a_straight_guide_to_apis_and_architecture_concepts.md

```diff
@@ -0,0 +1,102 @@
+---
+title: "A straight guide to APIs and architecture concepts"
+subtitle: From a newcomer experience
+date: 2022-11-29T06:00:00+01:00
+type: Communication
+tags:
+  - APIs
+  - Architectures
+cover: /img/posts/a_straight_guide_to_APIs_and_architecture_concepts/cover.webp
+authors:
+  - name: Barbaño González
+    photo: /img/avatars/barbano.webp
+    link: https://www.linkedin.com/in/barbano-gonzalez-moreno
+excerpt: Basic concepts about APIs and architectures for beginners and non-developers.
```

Comment:
```suggestion
excerpt: Basic concepts about APIs and architectures for beginners and non-developers.
featured: true
```

so it will show up on top